contracts/staking/compiler.json

```diff
@@ -0,0 +1,21 @@
+{
+    "artifactsDir": "generated-artifacts",
+    "contractsDir": "contracts",
+    "useDockerisedSolc": false,
+    "compilerSettings": {
+        "evmVersion": "constantinople",
+        "optimizer": { "enabled": true, "runs": 1000000 },
```

Comment:
I believe we have an updated version of `compiler.json` with more specific optimizer settings.

Reply:
Good eye, I've updated the `erc1155` package's compiler.json as well 🙏